setupbase.py

```diff
@@ -17,8 +17,10 @@
 
 from urllib.request import urlopen
 
-import distutils.cmd
-import distutils.log
+try:
+    from setuptools import Command
+except ImportError:
+    from distutils.cmd import Command
```

Comment:
Maybe we should define a requirement for a minimum `setuptools` version to prevent having to do a fallback to `distutils`